_data/toc/page-builder.yml

```diff
@@ -141,6 +141,34 @@ pages:
     url: /page-builder/docs/how-to/how-to-deactivate-pagebuilder.html
     versionless: true
 
+- label: BlueFoot content migration
+  versionless: true
+  children:
+
+  - label: Overview of content migration
+    url: /page-builder-migration/docs/
```

Comment:
should it be `/page-builder-migration/docs/index.html`?

Reply:
@belbiy recommended without the file name.